pypowerwall/fleetapi/fleetapi.py

```diff
@@ -51,16 +51,19 @@
 # FleetAPI Class
 
 import os
-import json 
-import requests
+import json
 import logging
 import sys
-import urllib.parse
 import time
+import urllib.parse
+import requests
 
 # Defaults
 CONFIGFILE = ".pypowerwall.fleetapi"
 SCOPE = "openid offline_access energy_device_data energy_cmds"
+SETUP_TIMEOUT = 15   # Time in seconds to wait for setup related API response
+REFRESH_TIMEOUT = 60 # Time in seconds to wait for refresh token response
+API_TIMEOUT = 10     # Time in seconds to wait for FleetAPI response
```

Comment:
Raising the default FleetAPI timeout from 5s to 10s. And, using the default = 10s may not be enough time to adequately handle the FleetAPI calls for setup of token refresh so setting higher limits for those.